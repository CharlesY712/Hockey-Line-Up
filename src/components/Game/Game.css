section {
  margin-top: 100px;
  height: 70vh;
  overflow-y: scroll
}

h1 {
  font-size: .8rem;
  margin: 0px;
}

.game-box {
  box-sizing: border-box;
  text-align: center;
  color: #A3A8AE;
  width: 94%;
  height: 4rem;
  background-color: rgb(60, 64, 70);
  padding: 1rem;
  margin: 0 auto 1rem auto;
}

@media (min-width: 414px) {
  section {
    margin-top: 150px;
    height: 68vh;
  }

  h1 {
    font-size: 1.11rem;
  }

  .game-box {
    height: 5rem;
  }
}

@media (min-width: 591px) {
  section {
    margin-top: 150px;
    height: 62vh;
  }

  h1 {
    font-size: 1.5rem;
  }

  .game-box {
    height: 6rem;
  }
}

@media (min-width: 768px) {
  section {
    margin-top: 175px;
    height: 81vh;
  }

  h1 {
    font-size: 2rem;
  }

  .game-box {
    height: 6.5rem;
  }
}

@media (min-width: 912px) {
  section {
    margin-top: 175px;
  }
  
  h1 {
    font-size: 2rem;
  }

  .game-box {
    height: 7rem;
  }
}

@media (min-width: 1056px) {
  h1 {
    font-size: 2rem;
  }

  .game-box {
    height: 7rem;
    width: 75vw;
  }
}
