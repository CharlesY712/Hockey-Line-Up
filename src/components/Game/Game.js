import React from 'react';
import PropTypes from 'prop-types';
import './Game.css';

const Game = ({
  homeTeamCity,
  homeTeamName,
  awayTeamCity,
  awayTeamName,
  time,
  date 
}) => {
  return (
    <article className="game-box">
      <h1>{awayTeamCity} {awayTeamName} vs
        {homeTeamCity} {homeTeamName} on {date} @ {time}</h1>
    </article>
  );
};

Game.propTypes = {
  homeTeamCity: PropTypes.string,
  homeTeamName: PropTypes.string,
  awayTeamCity: PropTypes.string,
  awayTeamName: PropTypes.string,
  noGames: PropTypes.string,
  date: PropTypes.string,
  time: PropTypes.string
};

export default Game;