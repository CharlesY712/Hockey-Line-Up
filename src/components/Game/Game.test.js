import React from 'react';
import Game from './Game';
import { shallow } from 'enzyme';

describe('Game', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Game/>);
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
