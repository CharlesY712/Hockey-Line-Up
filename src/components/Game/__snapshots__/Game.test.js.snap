// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Game />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1 />,
      "className": "game-box",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": undefined,
      },
      "ref": null,
      "rendered": null,
      "type": "h1",
    },
    "type": "article",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1 />,
        "className": "game-box",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h1",
      },
      "type": "article",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
